Add note about RAM issues to README.md (#282)
fortran-lang/stdlib: README.md

cmake --build build --target test
```

> 
> 

Author: ```suggestion
```

make -f Makefile.manual
```

> ## Memory issues during compilation
> 
> Some issues can be encountered during compilation on low RAM computers.
> These issues are due to the high number of procedures generated by the preprocessor.
> They can be avoided by limiting the maximum rank of generated procedures, as follows with `CMake`:

Reviewer: ```suggestion
## Limiting the maximum rank of generated procedures

Stdlib's preprocessor (fypp) by default generates specific procedures for arrays of all ranks, up to rank 15.
This can result in long compilation times and, on some computers, exceeding available memory.
If you know that you won't need all 15 ranks, you can specify the maximum rank for which the specific procedures will be generated.
For example, with CMake:
```

(notice that I don't use `monospace style` for proper names like CMake, stdlib, fypp, and such. Code style should be reserved for code, IMO)

```sh

> cmake -B build
> 
> cmake --build build -DCMAKE_MAXIMUM_RANK=4

Reviewer 2: ```suggestion
cmake -B build -DCMAKE_MAXIMUM_RANK=4

cmake --build build
```

Author: Indeed. I was too fast to copy my terminal... Thanks for the review @awvwgk .

> cmake -B build -DCMAKE_MAXIMUM_RANK=4
> 
> cmake --build build
> 
> cmake --build build --target test

Reviewer: Whitespace here seemed unnecessary, but you decide.

```suggestion
cmake -B build -DCMAKE_MAXIMUM_RANK=4
cmake --build build
￼cmake --build build --target test
```

```
or as follows with `make`:

```sh
make -f Makefile.manual FYPPFLAGS=-DMAXRANK=4
```

> 

Reviewer: ```suggestion
Note that currently the minimum value for maximum rank is 4.

```
